elements/pf-chip/pf-chip.ts

```diff
@@ -55,24 +55,23 @@ export class PfChip extends LitElement {
   render() {
     return this.overflowChip ? html`
       <button id="outer">
-        <span class="chip-content">
-          <slot id="chip-text" part="text"></slot>
+        <span part="text">
+          <slot></slot>
         </span>
       </button>
     ` : html`
       <div id="outer">
-        <span class="chip-content">
-          <slot id="chip-text" part="text"></slot>
+        <span part="text">
+          <slot></slot>
         </span>
-        <button id="close-button"
-                ?hidden="${this.readonly || this.overflowChip}"
-                @click="${this.#onClick}"
-                aria-describedby="chip-text"
-                aria-label="${this.accessibleCloseLabel}">
-          <svg aria-hidden="true" fill="currentColor" viewBox="0 0 352 512">
-            <path d="M242.72 256l100.07-100.07c12.28-12.28 12.28-32.19 0-44.48l-22.24-22.24c-12.28-12.28-32.19-12.28-44.48 0L176 189.28 75.93 89.21c-12.28-12.28-32.19-12.28-44.48 0L9.21 111.45c-12.28 12.28-12.28 32.19 0 44.48L109.28 256 9.21 356.07c-12.28 12.28-12.28 32.19 0 44.48l22.24 22.24c12.28 12.28 32.2 12.28 44.48 0L176 322.72l100.07 100.07c12.28 12.28 32.2 12.28 44.48 0l22.24-22.24c12.28-12.28 12.28-32.19 0-44.48L242.72 256z"></path>
-          </svg>
-        </button>
+        <pf-button id="close-button"
+                   plain
+                   icon="close"
+                   icon-set="patternfly"
+                   label="${this.accessibleCloseLabel}"
+                   aria-describedby="chip-text"
```

Comment:
This is an accessibility issue that @hellogreg and I reverted back to button because that was the way to solve cross-root aria issues. We need the `aria-describedby` to be a working IDRef to the chip text in order to be compatible with all assistive tech. This was the one issue that you and I had discussed last week that if we reverted all the other cross-root aria stuff would not be deal breakers.

Reply:
24e5ca26 should hopefully fix this